src/main/java/com/shopping_app/shoppingApp/model/User/Request/UserUpdateRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.shopping_app.shoppingApp.model.User.Request;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import javax.validation.constraints.NotBlank;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Builder
+@Setter
+public class UserUpdateRequest {
+
+    @NotBlank
+    private String name;
+
+    @NotBlank
+    private String email;
```

Comment:
Can you update the email for an user in an e-commerce site?

Reply:
Sorry my bad haven't noticed, sure will update this

Reply:
done